Ginger/Ginger/ALM/ALMConnectionPage.xaml.cs

```diff
@@ -143,52 +170,36 @@ private void SetControls()
             }
         }
 
-        public void RefreshALMSolutionSettings()
+        private void ChangeALMType()
         {
-            if ((ALMIntegration.eALMType) WorkSpace.Instance.Solution.AlmType == ALMIntegration.eALMType.QC && QCRadioButton.IsChecked == false)
+            if (CurrentAlmConfigurations.AlmType == GingerCoreNET.ALMLib.ALMIntegration.eALMType.QC && QCRadioButton.IsChecked != null ? (bool)QCRadioButton.IsChecked : false)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)